<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QVBoxLayout" name="drop_shadow_layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QFrame" name="Content">
      <property name="minimumSize">
       <size>
        <width>1080</width>
        <height>780</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="drop_shadow_frame">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>1080</width>
         <height>780</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>1080</width>
         <height>780</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="type_1">
        <property name="geometry">
         <rect>
          <x>84</x>
          <y>190</y>
          <width>250</width>
          <height>400</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>400</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>400</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(168, 255, 120, 255), stop:1 rgba(120, 255, 214, 255));

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QFrame" name="progress_bar_1">
         <property name="geometry">
          <rect>
           <x>35</x>
           <y>55</y>
           <width>180</width>
           <height>180</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="frame">
          <property name="geometry">
           <rect>
            <x>15</x>
            <y>40</y>
            <width>150</width>
            <height>80</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame{
	border: None;
	image: url(./app/component/stink.png);
	background-color:None;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QLabel" name="org_percent">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>120</y>
            <width>110</width>
            <height>40</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: None;
border: None;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;25%&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>25</x>
           <y>320</y>
           <width>200</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: None</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;ORGANIC&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="type_2">
        <property name="geometry">
         <rect>
          <x>414</x>
          <y>190</y>
          <width>250</width>
          <height>400</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>400</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>400</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(168, 255, 120, 255), stop:1 rgba(120, 255, 214, 255));

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QFrame" name="progress_bar_2">
         <property name="geometry">
          <rect>
           <x>35</x>
           <y>55</y>
           <width>180</width>
           <height>180</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="frame_7">
          <property name="geometry">
           <rect>
            <x>15</x>
            <y>40</y>
            <width>150</width>
            <height>80</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame{
	border: None;
	background-color:None;
	image: url(./app/component/plastic.png);
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QLabel" name="inorg_percent">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>120</y>
            <width>110</width>
            <height>40</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: None;
border: None;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;25%&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>320</y>
           <width>220</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: None</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;INORGANIC&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="type_3">
        <property name="geometry">
         <rect>
          <x>744</x>
          <y>190</y>
          <width>250</width>
          <height>400</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>400</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>400</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(168, 255, 120, 255), stop:1 rgba(120, 255, 214, 255));

</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QFrame" name="progress_bar_3">
         <property name="geometry">
          <rect>
           <x>35</x>
           <y>55</y>
           <width>180</width>
           <height>180</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>180</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="frame_9">
          <property name="geometry">
           <rect>
            <x>15</x>
            <y>40</y>
            <width>150</width>
            <height>80</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame{
	border: None;
	image: url(./app/component/other.png);
	background-color:None;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QLabel" name="other_percent">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>120</y>
            <width>110</width>
            <height>40</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>110</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI Black</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: None;
border: None;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;25%&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </widget>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>320</y>
           <width>150</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: None</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;OTHERS&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="stat_frame">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>660</y>
          <width>200</width>
          <height>60</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(168, 255, 120, 255), stop:1 rgba(120, 255, 214, 255));
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>200</width>
           <height>60</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>STATISTIC</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QFrame" name="Chart">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>1080</width>
         <height>780</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(168, 255, 120, 255), stop:1 rgba(120, 255, 214, 255));

border-radius: 15px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>315</x>
          <y>20</y>
          <width>440</width>
          <height>100</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>48</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="text">
         <string>STATISTIC</string>
        </property>
       </widget>
       <widget class="QPushButton" name="exit_btn">
        <property name="geometry">
         <rect>
          <x>1030</x>
          <y>20</y>
          <width>30</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(245, 47, 47);
border-radius: 15px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QWidget" name="chart_widget" native="true">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>180</y>
          <width>980</width>
          <height>550</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
</string>
        </property>
       </widget>
       <widget class="QWidget" name="Toggle_Btn" native="true">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>130</y>
          <width>100</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border: 2px solid rgb(100, 107, 96);</string>
        </property>
        <widget class="QWidget" name="pos_1" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>50</width>
           <height>40</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(168, 255, 120, 255), stop:1 rgba(120, 255, 214, 255));
border: solid rgba(255, 255, 255, 0);
margin: 2px;</string>
         </property>
        </widget>
        <widget class="QLabel" name="Month">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>0</y>
           <width>50</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
</string>
         </property>
         <property name="text">
          <string>M</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="Day">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>50</width>
           <height>40</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
</string>
         </property>
         <property name="text">
          <string>D</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <zorder>pos_1</zorder>
        <zorder>Day</zorder>
        <zorder>Month</zorder>
       </widget>
       <widget class="QDateEdit" name="dateEdit">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>130</y>
          <width>130</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
</string>
        </property>
       </widget>
      </widget>
      <zorder>Chart</zorder>
      <zorder>drop_shadow_frame</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="test.qrc"/>
 </resources>
 <connections/>
</ui>
